### PR TITLE
Adding information about misc ppa

### PR DESCRIPTION
**Pull request by shortcipher3**
# Adding documentation to make misc ppa more discoverable

I didn't find any documentation about the misc ppa on my first pass through the examples, so I added some.

### CODE REVIEW

File: Documentation/getting-started-ubuntu-ppa.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ $ sudo apt install nnstreamer
 * nnstreamer-dev : Required to build C function-filters and to build your own nnstreamer plugins.
 * nnstreamer-edgetpu : Allows to use edge-TPU in a pipeline.
 * nnstreamer-flatbuf : Allows to convert-from and decode-to flatbuf streams.
+* nnstreamer-misc: Allows to use join plugin.
```
Comment (gichan-jang) [inline Documentation/getting-started-ubuntu-ppa.md:25]: Only ` join` is included now in `nnstreamer-misc` package, but other plugins may be added in the future.
How about changing like this?
```suggestion
* nnstreamer-misc: Provides additional gstreamer plugins for nnstreamer pipelines. Included plugins: join
```

File: Documentation/getting-started-ubuntu-ppa.md [hunk 2]
```diff
@@ -30,6 +31,10 @@ $ sudo apt install nnstreamer
 * nnstreamer-tensorflow : Allows to use TensorFlow models in a pipeline. (From tensorflow 1.13.1 by default)
 * nnstreamer-tensorflow-lite : Allows to use TensorFlow-lite models in a pipeline. (From tensorflow 1.13.1 by default)
 
+For a full list of plugins run:
+```
+lz4cat -d /var/lib/apt/lists/ppa.launchpad.net_nnstreamer_ppa_ubuntu_dists_*_Packages.lz4 | grep "Package:"
```
Comment (gichan-jang) [inline Documentation/getting-started-ubuntu-ppa.md:36]:
```suggestion
lz4cat -d /var/lib/apt/lists/ppa.launchpad.net_nnstreamer_ppa_ubuntu_dists_*_Packages | grep "Package:"
```

Comment (myungjoo) [inline Documentation/getting-started-ubuntu-ppa.md:36]: In general, ```$ apt-cache search nnstreamer``` will do what you intended to do.
Besides, in my Ubuntu machine, this Packages file is not compressed.

Comment (gichan-jang) [reply Documentation/getting-started-ubuntu-ppa.md:36]: myungjoo's suggestion looks better.

Comment (shortcipher3) [reply Documentation/getting-started-ubuntu-ppa.md]: I like this suggestion, just wish it also included all of the other packages like `ssat`.

**Comment by gichan-jang** [comment]
Instead of a new commit for the modifications, let's merge them with the previous commit.
There is no need for two commits.

**Comment by myungjoo** [comment]
> Instead of a new commit for the modifications, let's merge them with the previous commit.
> There is no need for two commits.

Don't worry, I can squash-merge.

**Review by myungjoo — APPROVED** [review]
LGTM.

@shortcipher3 Please squash the commits.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**